rr_rviz_plugins/include/rr_rviz_plugins/SpeedometerPanel.h

```diff
@@ -0,0 +1,71 @@
+#ifndef PROJECT_SPEEDOMETERPANEL_H
+#define PROJECT_SPEEDOMETERPANEL_H
+
+#include <ros/ros.h>
+#include <rviz/panel.h>
+#include <rr_platform/chassis_state.h>
+#include <rr_platform/steering.h>
+#include <QLabel>
+#include <QtGui>
+#include <QPainter>
+#include <math.h>
+#include <QString>
+
+
+/*
```

Comment:
maintainer preference but I would remove the comments that came with the example panel now that you implemented something.

Reply:
Yes, @mukundSai ^